defs/support.json

```diff
@@ -1,100 +1,245 @@
 {
-	"trezor1": {
-		"Bitcoin": "1.5.2",
-		"Testnet": "1.5.2",
-		"Bcash": "1.6.2",
-		"Bcash Testnet": "1.6.2",
-		"Bgold": "1.6.2",
-		"Bprivate": "1.6.2",
-		"Dash": "1.5.2",
-		"Dash Testnet": "1.6.2",
-		"Decred": "1.6.2",
-		"Decred Testnet": "1.6.2",
-		"DigiByte": "1.6.0",
-		"Dogecoin": "1.5.2",
-		"Fujicoin": "1.6.1",
-		"Groestlcoin": "1.6.2",
-		"Groestlcoin Testnet": "1.6.2",
-		"Litecoin": "1.5.2",
-		"Litecoin Testnet": "1.6.2",
-		"Monacoin": "1.6.0",
-		"Namecoin": "1.5.2",
-		"Vertcoin": "1.6.1",
-		"Viacoin": "1.6.2",
-		"Zcash": "1.6.2",
-		"Zcash Testnet": "1.6.2",
-		"Zcoin": "1.6.2",
-		"Zcoin Testnet": "1.6.2"
-	},
-	"trezor2": {
-		"Bitcoin": "2.0.5",
-		"Testnet": "2.0.5",
-		"Bcash": "2.0.7",
-		"Bcash Testnet": "2.0.7",
-		"Bgold": "2.0.7",
-		"Bprivate": "2.0.7",
-		"Dash": "2.0.5",
-		"DigiByte": "2.0.5",
-		"Dogecoin": "2.0.5",
-		"Fujicoin": "2.0.5",
-		"Groestlcoin": "2.0.8",
-		"Groestlcoin Testnet": "2.0.8",
-		"Litecoin": "2.0.5",
-		"Litecoin Testnet": "2.0.7",
-		"Monacoin": "2.0.5",
-		"Namecoin": "2.0.5",
-		"Vertcoin": "2.0.5",
-		"Viacoin": "2.0.7",
-		"Zcash": "2.0.7",
-		"Zcash Testnet": "2.0.7",
-		"Zcoin": "2.0.7",
-		"Zcoin Testnet": "2.0.7",
-		"Zencash": "2.0.8"
-	},
-	"connect": {
-		"Bitcoin": true,
-		"Testnet": true,
-		"Bcash": true,
-		"Bgold": true,
-		"Dash": true,
-		"Litecoin": true,
-		"Zcash": true
-	},
-	"webwallet": {
-		"Bitcoin": true,
-		"Testnet": true,
-		"Bcash": true,
-		"Bgold": true,
-		"Bprivate": true,
-		"Dash": true,
-		"Decred": true,
-		"Decred Testnet": true,
-		"DigiByte": true,
-		"Dogecoin": true,
-		"Fujicoin": true,
-		"Groestlcoin": true,
-		"Groestlcoin Testnet": true,
-		"Litecoin": true,
-		"Litecoin Testnet": true,
-		"Monacoin": true,
-		"Vertcoin": true,
-		"Viacoin": true,
-		"Zcash": true,
-		"Zcash Testnet": true,
-		"Zcoin": true
-	},
-	"electrum": {
-		"Bitcoin": "https://electrum.org",
-		"Bcash": "https://electroncash.org",
-		"Bgold": "https://github.com/BTCGPU/electrum",
-		"Bprivate": "https://github.com/BTCPrivate/electrum-btcp",
-		"Dash": "https://electrum.dash.org",
-		"Groestlcoin": "https://www.groestlcoin.org/groestlcoin-electrum-wallet/",
-		"Koto": "https://electrum.kotocoin.info",
-		"Litecoin": "https://electrum-ltc.org",
-		"Monacoin": "https://electrum-mona.org",
-		"Namecoin": "https://github.com/namecoin/electrum-nmc",
-		"Viacoin": "https://vialectrum.org",
-		"Zcoin": "https://github.com/zcoinofficial/electrum-xzc"
-	}
+    "btc:BCH": {
```

Comment:
Maybe use prefix `coin:` instead of `btc`? I understand why you chose that, but I think `coin` is less confusing.